packages/core/src/ElementsFactory/interfaces.ts

```diff
@@ -24,6 +24,7 @@ export interface FormProps extends React.HTMLAttributes<HTMLFormElement> {
 export interface LoaderProps extends React.HTMLAttributes<HTMLElement> {
   inline?: boolean;
   center?: boolean;
+  color?: 'primary' | 'secondary' | 'inherit';
```

Comment:
use variant for 'primary', 'secondary'...  not color. should be the same as ButtonProps.